bitbots_lowlevel/bitbots_ros_control/scripts/pose_check.py

```diff
@@ -0,0 +1,445 @@
+#!/usr/bin/env python3
+
+"""
+Moves the robots to a bunch of different poses to check if the robot is configured/build/calibrated as expected
+"""
+
+import math
+import os
+
+import rclpy
+import rclpy.publisher
+from ament_index_python import get_package_share_directory
+from colorama import Back, Fore, Style
+from rclpy.node import Node
+from rclpy.publisher import Publisher
+
+from bitbots_msgs.msg import HeadMode, JointCommand
+
+SPEED = 1.0
+DYNAMIXEL_CMD_TOPIC = "/DynamixelController/command"
+JOINT_NAMES = [
+    "HeadPan",
+    "HeadTilt",
+    "LShoulderPitch",
+    "LShoulderRoll",
+    "LElbow",
+    "RShoulderPitch",
+    "RShoulderRoll",
+    "RElbow",
+    "LHipYaw",
+    "LHipRoll",
+    "LHipPitch",
+    "LKnee",
+    "LAnklePitch",
+    "LAnkleRoll",
+    "RHipYaw",
+    "RHipRoll",
+    "RHipPitch",
+    "RKnee",
+    "RAnklePitch",
+    "RAnkleRoll",
+]
+
+
+def num_to_emoji(num: int) -> str:
+    emoji_numbers = ["0️⃣", "1️⃣", "2️⃣", "3️⃣", "4️⃣", "5️⃣", "6️⃣", "7️⃣", "8️⃣", "9️⃣"]
+    return f'{"".join([emoji_numbers[int(digit)] for digit in str(num)])} '
```

Comment:
```suggestion
    return f'{"".join([emoji_numbers[int(digit)] for digit in str(num)])}'
```
Is this last space intentional?

Reply:
Yes. This is also the reason for the string around it. Otherwise funny utf8 magic happens and the next character is included in the number.